airsonic-main/src/test/java/org/airsonic/player/TestCaseUtils.java

```diff
@@ -16,6 +18,8 @@
 
 public class TestCaseUtils {
 
+    private static final Logger logger = LoggerFactory.getLogger(TestCaseUtils.class);
```

Comment:
The majority of the rest of the codebase seems to use `LOG` as the logger variable rather than `logger`.  I don't really care which it is, but for the sake of consistency, would it be better to stick with the majority convention?

Reply:
Probably - `logger` is just a sub conscious habit for me :) Might need to add a checkstyle so we can enforce.

Reply:
Proposal: Build some custom checkstyles. See https://github.com/airsonic/checkstyle. Only gotcha is that we would probably need to publish this to the maven central.

Reply:
I like the checkstyle approach, that is a good way to enforce going forward!